.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,31 @@
+name: algolia-scraper
+on:
```

Comment:
You're closer, but this is still wrong.  We want to run a series of `jobs` whenever we push to the main branch.  One of those jobs is to build the site, another is to index it, another is to deploy it.  The order matters, so we need them to all go in a single workflow, and depend on each other.  Your job needs to go in the same development.yml file that we already started, and be one part of the deploy step.

Reply:
i see so same file i was like how do u call one from another

Reply:
![image](https://user-images.githubusercontent.com/55400510/142686048-afc0a100-bdcc-403a-8e32-fd4e80c0b5bd.png)

Reply:
oh i think u combine run and env parts